Session/Meta/AppDelegate.swift

```diff
@@ -132,24 +136,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         
         // NOTE: Fix an edge case where user taps on the callkit notification
         // but answers the call on another device
-        stopPollers(shouldStopUserPoller: !self.hasIncomingCallWaiting())
+        stopPollers(shouldStopUserPoller: !self.hasCallOngoing())
         
         // Stop all jobs except for message sending and when completed suspend the database
         JobRunner.stopAndClearPendingJobs(exceptForVariant: .messageSend) {
-            NotificationCenter.default.post(name: Database.suspendNotification, object: self)
+            if !self.hasCallOngoing() {
+                NotificationCenter.default.post(name: Database.suspendNotification, object: self)
+            }
         }
     }
     
-    func applicationDidReceiveMemoryWarning(_ application: UIApplication) {
-        Logger.info("applicationDidReceiveMemoryWarning")
-    }
-    
-    func applicationWillTerminate(_ application: UIApplication) {
-        DDLog.flushLog()
-        
-        stopPollers()
-    }
```

Comment:
Do we not want to do this on terminate anymore?

Reply:
Yep, I think I accidentally removed them...